src/yaml-support/yaml-schema.ts

```diff
@@ -0,0 +1,165 @@
+import * as _ from 'lodash';
+import Uri from 'vscode-uri';
+import * as vscode from 'vscode';
+import { yamlLocator } from "./yaml-locator";
+import { VSCODE_YAML, KUBERNETES_SCHEMA, KUBERNETES_GROUP_VERSION_KIND, GROUP_VERSION_KIND_SEPARATOR } from "./yaml-constant";
+
+// the schema for kubernetes
+const _definitions = {};
+
+// load the kubernetes schema and make some modifications to $ref node
+export function loadSchema(schemaFile: string) {
+    const schemaRaw = require(schemaFile);
```

Comment:
Is `require` idiomatic?  I like the conciseness, but I usually think of `require` as loading a code module.

Reply:
ok, require will have bad side effect of loading js rather than json.